Dockerfile.postgres

```
@@ -0,0 +1,37 @@
+FROM python:3.9 AS decrypt
+
+COPY ./secrets /secrets
+ARG CRYPT_PASSWORD
```

Lets fail build process if CRYPT_PASSWORD is not set